src/HealthCheckServiceProvider.php

```diff
@@ -24,6 +25,10 @@ public function boot()
             return new AppHealth($checks);
         });
 
+        $this->commands([
```

Comment:
I'm not sure if we need to add this code block to make sure we only register commands when the app runs in console.

```php
if ($this->app->runningInConsole()) {
    // $this->commands([...])
}
```

Reply:
I'd seen both, checked Laravel docs though and they include the if statement so I'll add it